docs/querying/sql-window-functions.md

```diff
@@ -88,40 +88,171 @@ You can use the OVER clause to treat other Druid aggregation functions as window
 
 Window functions support aliasing.
 
-## Define a window with the OVER clause
+## Window function syntax
+
+In general, Druid Window functions use the following syntax:
+
+```sql
+SELECT
+    dimensions,
+    aggregation function(s)
+    window_function()
+      OVER ( PARTITION BY partitioning expression
+             ORDER BY order expression
+             frame clause
+            )
+    FROM table
+    GROUP BY dimensions
+```
 
 The OVER clause defines the query windows for window functions as follows:
 - PARTITION BY indicates the dimension that defines the rows within the window
 - ORDER BY specifies the order of the rows within the windows.
 
+for example, the following OVER clause example sets the window dimension to `channel` and orders the results by the absolute value of `delta` ascending:
+
+```sql
+...
+RANK() OVER (PARTITION BY channel ORDER BY ABS(delta) ASC)
+...
+```
+
+Druid applies the GROUP BY dimensions first before calculating all non-window aggregation functions. Then it applies the window function over the aggregate results.
```

Comment:
Should also specify what an empty OVER() indicates

Reply:
@soumyava, @kgyrtkirk what does an empty `OVER()` indicate?

Reply:
empty OVER() or absence of a partition by indicates that all the data belong to a single window